packages/react/src/hooks/contracts/useContractInfiniteReads.ts

```diff
@@ -95,7 +95,11 @@ export function useContractInfiniteReads<
   select,
   staleTime,
   suspense,
-}: UseContractInfiniteReadsConfig<TContracts, TPageParam>) {
+}: UseContractInfiniteReadsConfig<
```

Comment:
Are these changes from another PR?

Reply:
Yup! Let me pull in from main so it's not cluttering the diff here.